src/System.Security.Cryptography.Xml/tests/XmlDecryptionTransformTest.cs

```diff
@@ -134,6 +134,7 @@ public void LoadInnerXml_XmlValidUriForExcept()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "https://github.com/dotnet/corefx/issues/18690")]
```

Comment:
What's the issue here? It's not at all apparent from the suppression tag.  Is this 17020?  The purpose of 18690 was to catch up with changes we'd already made that weren't port-consider.  Since 17020 is port-consider it probably makes sense to just tag this as that.